spec/features/drivers/user_can_create_new_driver_spec.rb

```diff
@@ -38,7 +38,7 @@
     expect(newly_created_driver.email_address).to eq('email@email.com')
     expect(newly_created_driver.age).to eq(18)
     expect(newly_created_driver.notes).to eq('notes')
-    expect(newly_created_driver.status).to eq(true)
+    expect(newly_created_driver).to be_truthy
```

Comment:
```suggestion
    expect(newly_created_driver.status).to be_truthy
```
status is still required, otherwise we are just checking if a newly_created_driver object is true.